src/olympia/addons/migrations/0047_auto_20230906_1220.py

```diff
@@ -0,0 +1,37 @@
+# Generated by Django 4.2.3 on 2023-09-06 12:20
+
+from django.db import migrations
+
+
+ANDROID_STATICTHEME_CAT_IDS = {
+    'abstract': 400,
+    'causes': 420,
+    'fashion': 424,
+    'film-and-tv': 426,
+    'firefox': 408,
+    'foxkeh': 410,
+    'holiday': 428,
+    'music': 422,
+    'nature': 402,
+    'other': 414,
+    'scenery': 406,
+    'seasonal': 412,
+    'solid': 418,
+    'sports': 404,
+    'websites': 416,
+}.values()
+
+
+def delete_android_static_theme_addon_category(apps, schema_editor):
+    AddonCategory = apps.get_model('addons', 'AddonCategory')
+    AddonCategory.objects.filter(category_id__in=ANDROID_STATICTHEME_CAT_IDS).delete()
```

Comment:
mmm that's a 500K rows `DELETE`... I think I would rather have that as a cleanup command that does it in chunks than something that blocks the push.

Reply:
It's harmless to keep them there but we could spin it out into a general clean-up for all old categories (or do it with/after #5989)

Reply:
Yeah drop a note in that issue, leave them for now.